components/framework/index.js

```diff
@@ -116,6 +116,12 @@ class ServerlessFramework {
     this.context.successProgress('removed');
   }
 
+  async package() {
+    this.context.startProgress('packaging');
+
+    await this.exec('serverless', ['package']);
```

Comment:
Just noticed that the progress is probably never turned into a success, did you notice that when testing?

It might be necessary to switch to this:

```suggestion
    await this.exec('serverless', ['package']);
    
    this.context.successProgress('packaged');
```

Reply:
Good spot! I also noticed I missed the options object which was actually causing a failure. 

<img width="394" alt="image" src="https://user-images.githubusercontent.com/14273378/181531435-1d22d6a9-bf96-4da0-932c-df2cceacccae.png">